packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -107,6 +107,7 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     <Stack className={mergedRootStyle}>
       <div className={mergedTextContainerStyle}>
         <TextField
+          tabIndex={1}
```

Comment:
Hmm, if these are global, how does it impact a composite embedded in an App? Will Contoso be able to set the necessary tabIndex for their site or will we have "stolen" tabIndex: 1 from them ?

Reply:
if they set their tabindex to 1 on their forms if there is one on the page with the composite it will just follow the same behavior as default. I will look into if I can remove them and get it to behave correctly without them though because they should not have to worry about that.

Reply:
should be tabIndex={0} instead, right?